components/lists/FeedList.tsx

```diff
@@ -8,8 +16,90 @@ import { View } from 'react-native';
  * - add tests for the feed list component
  */
 
+interface FeedListItemProps {
+  item: Feed;
+}
+
+const { height } = Dimensions.get('screen');
+
+const FeedListItem = ({ item }: FeedListItemProps) => {
+  const { colors } = useTheme();
+  return (
+    <View style={styles.feed} testID="feedItem">
+      <StyledText size="md">{item.body}</StyledText>
+      <View style={styles.imageContainer}>
+        <Image style={styles.image} source={{ uri: item.image }} />
```

Comment:
The images take long to load, we should use [`expo-image`](https://docs.expo.dev/versions/latest/sdk/image/) here instead, which loads faster and caches the images, but also make use of its `blurhash` to show a placeholder image while the image is loading. This is already installed in the project if you update this branch from `develop`.

Reply:
yes, and the `blurhash` has already been set in the `constants.ts` file.